tests/src/test/scala/whisk/core/controller/test/ActivationsApiTests.scala

```diff
@@ -72,7 +75,7 @@ class ActivationsApiTests extends ControllerTestCommon with WhiskActivationsApi
         activations foreach { put(activationStore, _) }
         waitOnView(activationStore, namespace, 2)
         whisk.utils.retry {
-            Get(s"$collectionPath") ~> sealRoute(routes(creds)) ~> check {
+            Get(s"$collectionPath") ~> Route.seal(routes(creds)) ~> check {
```

Comment:
i'm ok with the change - just have to carefully check all the changes to make sure it's not changing the tests (found 1 test difference on a cursory scan).